test/integration/api_paths_test.go

```diff
@@ -27,6 +27,7 @@ func TestRootAPIPaths(t *testing.T) {
 		"/logs/": http.StatusNotFound,
 	}
 
+	defer testutil.RequireEtcd(t).Terminate(t)
```

Comment:
Does this evaluate the first function immediately, and the second function deferred?